src/include/daos_srv/bio.h

```diff
@@ -696,4 +696,9 @@ int bio_replace_dev(struct bio_xs_context *xs, uuid_t old_dev_id,
  */
 int bio_set_led_state(struct bio_xs_context *xs, uuid_t devid,
 		      const char *led_state, bool reset);
+
+/*
+ * Get max bio chunk cnt for the xstream.
+ */
+int bio_get_max_chk_cnt();
```

Comment:
(style)  Bad function definition - int bio_get_max_chk_cnt() should probably be int bio_get_max_chk_cnt(void)